data/free worlds checkmate.txt

```diff
@@ -2627,6 +2627,7 @@ mission "FWC End"
 		event "syndicate tech available"
 		set "main plot completed"
 		set "free worlds plot completed"
+		event "Tarazed Graffias non-neutrality" 7
```

Comment:
To avoid double-applying the event, maybe just take these lines out any *only* have it trigger through the "patch" mission.